### PR TITLE
Fix spellings in history2.html

### LINKED ISSUE #50: Fix spellings in history2.html
**Is your feature request related to a problem?**
Found a spelling mistake in history2.html file

**Describe the solution you'd like**
Correct the spelling mistake found in the above file

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by gajindubandara**
## Purpose

The purpose of this PR is to fix #50

## Goals
Correct spelling mistakes in history2.html

## Approach
Remove unwanted letters 

### Screenshots
Screenshot (33)

  

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: history2.html [hunk 1]
```diff
@@ -195,7 +195,7 @@ <h1 class="text-center"><small>&nbsp;</small></h1>
...
...align="justify">><span class="dropcap">I</span>n the year 1906, Fr. Craner was made to relinquish the post of Principal, which he did with a deep sense of sadness, but with a feeling of discipline and obedience, as his services were needed elsewhere. He had already groomed his successor, Fr. D. Philip Caspersz OSB who was already a member of the teaching staff. After Fr. Caspersz assumed duties as Principal, his brother, Fr. James came in as Boarding Prefect. These two brothers, in a comparatively short time changed the status of the school, making it a College. On 20th of December 1907, the Annual Distribution of Prizes was held for the first time on a grand scale. The Rt. Revd. Dr. C. Pagnani OSB - Bishop of Kandy was the Chief Guest, and along with Fr. D.A. Pancrazi OSB, he distributed 100 prizes amongst a total student population of 275. The ceremony was held in what was called the "Big College Hall" measuring 100' x 27'. The programme opened with a rendering of the chorus 'Over the Hill' by the College Choir and ended with chanting of the 'Papal Hymn' and National Anthem. 'Electricity &amp; Magnetism' was introduced as a subject of the 'Cambridge Classes' in this year.<br><br>
```
Comment (jayasanka-sack) [inline history2.html:198]: Thanks for sending this pr @gajindubandara ! 

Could you please remove the extra bracket here. 

`<p align="justify">><span...`  
it should be,
`<p align="justify"><span...`

Comment (gajindubandara) [reply history2.html:198]: It's already removed ?

Comment (jayasanka-sack) [reply history2.html:198]: Oh sorry!

Comment (jayasanka-sack) [reply history2.html:198]: Oh sorry my bad!

Comment (gajindubandara) [reply history2.html:198]: Thanks !

**Pull request merged**